ideasbox/templates/ideasbox/includes/footer.html

```diff
@@ -35,6 +19,9 @@
                 <input type="submit" value="{% trans 'Change' %}" />
             </form>
         </div>
+        <div class="col half">
+            <img src="../../../../static/ideasbox/img/logoBSF.png" />
```

Comment:
You need to use `static` templatetag (have a look at `base.html` to have an example).